bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/discovery/GroheOndusDiscoveryService.java

```diff
@@ -82,10 +79,10 @@ protected void startScan() {
             ThingUID bridgeUID = bridgeHandler.getThing().getUID();
             ThingUID thingUID = null;
             switch (appliance.getType()) {
-                case SenseGuardAppliance.TYPE:
+                case org.grohe.ondus.api.model.guard.Appliance.TYPE:
                     thingUID = new ThingUID(THING_TYPE_SENSEGUARD, appliance.getApplianceId());
                     break;
-                case SenseAppliance.TYPE:
+                case Appliance.TYPE:
```

Comment:
Given that they are two `Appliance` classes in the api I would suggest making the namespaces explicit.
```suggestion
                case org.grohe.ondus.api.model.sense.Appliance.TYPE:
```

Reply:
Agreed! :)